README.md

```diff
@@ -5,12 +5,12 @@ The purpose of this test is 3 fold:
 * Assess how comfortable you are with writing automation tests and the approach taken to accomplish it
 
 ## 📕 What you need to do:
-Using the testing site https://gorest.co.in/, write automation tests for the following operations:
-1. Create a user and verify that the user got created
-2. Update a user and verify that the user got updated
-3. Delete a user and verify that user got deleted
+Using the testing site https://gorest.co.in/, write automation tests for the following endpoints:
+1. Create a new user
+2. Update user details
+3. Delete user
```

Comment:
Do we need to mention get user endpoint (as that is what can be used for verification of the above operations) ?

Reply:
Improved language to provide more freedom.